frame/support/procedural/src/construct_runtime/expand/call.rs

```diff
@@ -62,6 +62,41 @@ pub fn expand_outer_dispatch(
 		pub enum Call {
 			#variant_defs
 		}
+		#[cfg(test)]
+		impl Call {
+			/// Return a list of the module names together with their size in memory.
+			pub fn sizes() -> &'static [( &'static str, usize )] {
+				use #scrate::dispatch::Callable;
+				use core::mem::size_of;
+				&[#(
+					(
+						stringify!(#pallet_names),
+						size_of::< <#pallet_names as Callable<#runtime>>::Call >(),
+					),
+				)*]
+			}
+
+			pub fn check_size(limit: usize) {
```

Comment:
Is this useful outside of testing? I'm thinking that it should have a `#[cfg(test)]` otherwise, might help with code size bloat (is it actually a problem?)

Reply:
It also feels like that this can be `const`, since it doesn't seem to be reliant on anything dynamic.

Reply:
Its `impl` block is already under that guard.